bar/90_contents/SyncEngine/sync.js

```diff
@@ -381,11 +380,7 @@ function(request){
                 personalEntityAccessor.create(exData);
                 syncCount++;
               } else {
-                return {
-                  status : 500,
-                  headers : {"Content-Type":"application/json"},
-                  body: [JSON.stringify({"error": e.message})]
-                };
+                return Response(500, '{"error" : ' + e.message + '}')
```

Comment:
Please use JSON.stringify inside Response. It is very hard to debug or add new message when it is explicitly written in string format.  

See [sample function](https://github.com/dixonsiu/app-uc-cell-creator-wizard/blob/master/src/unitService/__src/user_cell_create.js#L194).